# This message describes information in a .gen file produced by Avida analyze
# Info about genotype
string trial_id
string env_id
string genotype_id
# Genotype Data
int32 generation
float64 merit
int32 generation_length
float64 fitness
int16 errors
int16 genome_size
int16 copied_size
int16 executed_size
string offspring
string[] tasks
int32[] task_cnts
string genome
